x/committee/types/permissions.go

```diff
@@ -473,9 +473,10 @@ type AllowedAssetParam struct {
 }
 
 func (aap AllowedAssetParam) Allows(current, incoming bep3types.AssetParam) bool {
+
 	allowed := ((aap.Denom == current.Denom) && (aap.Denom == incoming.Denom)) && // require denoms to be all equal
 		((current.CoinID == incoming.CoinID) || aap.CoinID) &&
-		(current.Limit.Equal(incoming.Limit) || aap.Limit) &&
+		(current.SupplyLimit.Equals(incoming.SupplyLimit) || aap.Limit) &&
 		((current.Active == incoming.Active) || aap.Active)
 	return allowed
 }
```

Comment:
Continuing from https://github.com/Kava-Labs/kava/pull/616#discussion_r468220235
I feel like being able to change MaxSwapAmount and MinBlockLock would be useful if it's not too much work. Gives more options for tuning the deputy.